src/screens/Collectible/Collectible.js

```diff
@@ -97,20 +103,58 @@ const CollectibleImage = styled(CachedImage)`
 const iconSend = require('assets/icons/icon_send.png');
 const genericCollectible = require('assets/images/no_logo.png');
 
-class CollectibleScreen extends React.Component<Props> {
-  shouldComponentUpdate(nextProps: Props) {
+class CollectibleScreen extends React.Component<Props, State> {
+  state = {
+    isImageViewVisible: false,
+  };
+
+  shouldComponentUpdate(nextProps: Props, nextState: State) {
     const isFocused = this.props.navigation.isFocused();
     if (!isFocused) {
       return false;
     }
-    const isEq = isEqual(this.props, nextProps);
-    return !isEq;
+    if (!isEqual(this.state, nextState)) {
```

Comment:
Those could be combined: 
```suggestion
    if (!isEqual(this.state, nextState) || !isEqual(this.props, nextProps)) return true;
```

Reply:
Updated